server/conf/i18n/messages

```diff
@@ -241,6 +241,7 @@ validation.phoneCountryCodeRequired=Country must be selected
 validation.phoneNumberMustContainNumbersOnly=Phone number cannot contain non-number characters
 validation.invalidPhoneNumberProvided=This phone number is invalid
 validation.phoneMustBeLocalToCountry=The phone you have provided does not belong to the country
+validation.phoneNumberMustContainOnlyTenDigits=The phone number must contain only ten digits
```

Comment:
Question (blocking): Because we're adding a new string, do we need to flag-guard this so we don't show untranslated strings to users?

Reply:
what is flag-guard?

Reply:
I think Danielle answered this in slack but: I meant that there's some delay between adding the new string here and actually getting translations for it, so I'm concerned that during that in-between period, users may start seeing this error message not translated to their language.

I think the best approach from https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#internationalization-for-application-strings is to write a separate PR that adds the message, then submit this PR once all the message translations come through?

Reply:
Chatted offline, decided to re-use an existing error message so we don't need to wait for translations.